packages/block-editor/src/components/off-canvas-editor/appender.js

```diff
@@ -41,6 +45,43 @@ export const Appender = forwardRef( ( props, ref ) => {
 				selectBlockOnInsert={ false }
 				shouldDirectInsert={ false }
 				__experimentalIsQuick
+				renderToggle={ ( {
+					onToggle,
+					disabled,
+					isOpen,
+					hasSingleBlockType,
+					toggleProps = {},
+				} ) => {
+					const { onClick, ...rest } = toggleProps;
+					// Handle both onClick functions from the toggle and the parent component.
+					function handleClick( event ) {
+						if ( onToggle ) {
+							onToggle( event );
+						}
+						if ( onClick ) {
+							onClick( event );
+						}
+					}
+					return (
+						<Button
+							icon={ plus }
+							label={ __( 'Add menu item' ) }
+							tooltipPosition="bottom"
```

Comment:
Is this used? I do not see a tooltip present and because the button text is always visible I don't think it is needed?